### ISSUE TITLE
laps-runner.py ignores 'native-laps' option in laps-runner.json when it is set to false

### ISSUE THREAD

**Issue by fraunhofer-ipm-itk** [schorschii/LAPS4LINUX#21]
Hello,

I am currently trying to update laps-runner to a newer version and during that process I noticed the following issue:

**Description:** When 'native-laps' is set to false in the laps-runner.json config, the script `laps-runner.py` just ignores it and tries native Laps anyways:
```
Connected as: REDACTED None
Updating password (forced update)...
Using Native LAPS JSON format
Password and expiration date changed successfully in LDAP directory (attribute ms-Mcs-AdmPwd, new expiration REDACTED)
Password successfully changed in local database
```
(see attachments for the used config)

**Steps to reproduce:**
1. Install laps-runner and its' dependencies
2. Edit the default config so that laps-runner can sucessfully connect to a domain controller
3. change or add `"native-laps": false,` (alter the ldap-attribute parameters if you run into errors)
4. run `./laps-runner.py -c /path/to/your/config -f`

**Root cause:**
In the `LoadSettings` function of the laps-runer.py script, the native-laps option is treated as a string instead of a boolean:
```python
self.cfgUseNativeLapsAttributeSchema = str(cfgJson.get('native-laps', self.cfgUseNativeLapsAttributeSchema))
```
This causes it to be `"False"` instead of `False` (which gets interpreted as True by the check afterwards)

**Possible Solutions:**
1. Remove the "str" function call from line 306:
```python
self.cfgUseNativeLapsAttributeSchema = cfgJson.get('native-laps', self.cfgUseNativeLapsAttributeSchema)
```
3. Change the check to `if not(self.cfgUseNativeLapsAttributeSchema=="False"):`

Sorry if it is too little/too much, this is my first bug report.
If you have any further questions, feel free to ask :)

Kind regards,
Simon

### Attachments
used-configuration:
```json
{
    "server": [
        {
            "address": "REDACTED",
            "port": 636,
            "ssl": true
        },
        {
            "address": "REDACTED",
            "port": 636,
            "ssl": true
        }
    ],
    "domain": "REDACTED",
    "use-starttls":false,

 

    "security-descriptor": null,
    "native-laps": false,
    "ldap-attribute-password": "ms-Mcs-AdmPwd",
    "ldap-attribute-password-expiry": "ms-Mcs-AdmPwdExpirationTime",
    

 

    "cred-cache-file": "/tmp/laps.temp",
    "client-keytab-file": "/etc/krb5.keytab",

 

    "hostname": null,
    "password-change-user": "REDACTED",
    "password-days-valid": 1,
    "password-length": 10,
    "password-alphabet": "REDACTED"
}
```

**Comment by schorschii** [comment]
Hi, thanks for reporting this with detailed description and solution! I added your fix to the master branch (6436a810dfaff0aed15a5860bac4192acaf57f56). If everything works as expected now, I'll create a new release.

**Comment by fraunhofer-ipm-itk** [comment]
Hello,
sorry for the long wait. I managed test the fix ([6436a81](https://github.com/schorschii/LAPS4LINUX/commit/6436a810dfaff0aed15a5860bac4192acaf57f56)) today. It seems to work as expected for me. 
When setting `native-laps` to `false`, it is using legacy laps again.
Thank you :)
Kind regards,
Simon

**Comment by schorschii** [comment]
Thanks for your feedback. I released v1.7.2 and new installation packages now.
